### ISSUE TITLE
Provide methods for clearing, and specifically setting stats

### ISSUE THREAD

**Issue by jimeh** [jimeh/redistat#2]
Being able to remove statistics for certain labels and/or timeframes, and also setting them to a fixed value rather than only supporting incrementation could be useful for testing purposes.

**Comment by renemendoza** [comment]
Is it currently possible to set a specific date for an event? or to set an initial value for a counter?

**Comment by jimeh** [comment]
Yes, both are possible. Regarding specifying when an event happened, just pass in a Time object as the 3rd argument to the `Model#store` method.

Regarding setting an initial value, there's no specific set function at the moment, hence I created this very issue. However to just set an initial value, just trigger a normal event with your initial value. If an event with the same label hasn't been triggered before, it will increment the values from 0, resulting in what you're after.

I should get some more features done soon, and most importantly, some proper documentation.

**Comment by rainchen** [comment]
+1 for Provide methods for clearing

**Comment by rainchen** [comment]
any progress for this?

**Comment by lagsalot** [comment]
Herm... interesting project.  Is it dead?  Is there something better/currently maintained?

**Comment by jimeh** [comment]
Apologies for the silence. Redistat hasn't gotten much (or any really) of my attention for a while. Mostly cause I no longer have any active project which I maintain on a daily basis that's using it.

I will try to put aside some time for Redistat, and a few other projects of mine which haven't seen as much love as they deserve for some time.

In the meantime I'd be more than happy to at least review and accept any pull requests in the coming days.
